src/tabs/Tabs.tsx

```diff
@@ -176,8 +178,8 @@ export class Tabs extends PureComponent<TabsProps, TabsState> {
     }
 
     return (
-      <Fragment>
-        <div className={cc(['kirk-tabs', className, { 'kirk-tabs-fixed': isFixedTabs }])}>
+      <div className={cc([className])}>
```

Comment:
Just wondering: do you know the impact of changing the Fragment into a div? I remember we broke the trip cards in search with such a change few days ago. I guess here it's fine as it's not a dependancy on some other components, and you will update them with your PR on Kairos.

Reply:
I fear moving `className` to a `div` wrapping both the tabs and panels is a breaking change. That's what we use to have the tabs sticky (we don't want to stick the panels too)